File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for i in range(len(nums) - 1):
            for n in range(i + 1, len(nums)):
                num1 = nums[i]
                num2 = nums[n]
                if num1 + num2 == target:
                    return [i, n]
        return []

File: 0001-two-sum/0001-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target)
    {
      int[] originalIndices = Enumerable.Range(0, nums.Length).ToArray();

      Array.Sort(nums, originalIndices);

      int left = 0;
      int right = nums.Length - 1;

      while (left < right)
      {
        int currentSum = nums[left] + nums[right];

        if (currentSum == target)
        {
          return new int[] { originalIndices[left], originalIndices[right] };
        }
        else if (currentSum < target)
        {
          left++;
        }
        else
        {
          right--;
        }
      }

      return new int[0];
    }
}